script.js

```diff
@@ -0,0 +1,68 @@
+
+var start_answer = prompt("Hello, sweetie! I'm the Doctor. Doctor, who? Just the Doctor! I'm a Time Lord. Some kind of alien? Well, yes. I've got two hearts and a whole lot of angst. I travel through time and space in my machine called the TARDIS. Will you come with me? (Yes, or No)");
+
+  if (start_answer === "yes") { //YES
+
+
+    var travel_number = prompt("Fantastic! You shall be my companion. I love companions. Especially a sexy one such as yourself. Did I just say that out loud? Well! Let's get on with it then. To when would you prefer to go? \n-the past (1)\n-the present (2)\n-the future (3)");
+
+      if (travel_number === "1") { //PAST
+
+function ElizabethEra(){
+      alert(""Allons-y! I love the past. Mine is a bit dark and broodish, but you don't need to worry about that. Say, what do you think of the Elizabethan Era? It really is quite smashing.");
+    var pastoption = prompt("Whom would you rather meet? Queen Elizabeth or William Shakespeare? E or S").toUpperCase();
+          if (past_option === "E") {
+            QueenElizabeth();
+          }
+          else if (past_option === "S") {
+            WilliamShakespeare();
+          }
+}
+
+function QueenElizabeth(){
+      alert("Excellent! She loves me. But, who couldn't love a Time Lord? Oh dear, I just got a message in the TARDIS. She's in danger! It appears someone has devised a plot to assassinate her! She's a fixed point in time. If her history changes, time itself may be torn apart! We have to stop this. There are only two individuals who would possibly want to murder her: Mary Stuart, her cousin in Scotland who claims Elizabeth's throne is hers, or Phillip II, King of Spain. We only have time to intercept one individual.").toUpperCase();
+      var past_elizabeth = prompt("Who do you think is responsible for the plot? Mary Stuart, or Phillip of Spain? M or P?").toUpperCase();
+              if (past_elizabeth === "M") {
+                MaryStuart();
+              }
+              else if (past_option === "P") {
+                PhillipSpain
+              }
+}
+                var past_mary = prompt("MARY STUART.\nWell done, companion! It is Mary! As a prisoner in Scotland, she appears to have been secretly communicating with spies in England! Stick around, companion! I'm sure the Queen will reward you!");
```

Comment:
similar to the comment above, variables should not be snake_case, but instead of it should be camelCase :)